src/grpc_generated/go/go.mod

```diff
@@ -9,8 +9,8 @@ require (
 
 require (
 	github.com/golang/protobuf v1.5.2 // indirect
-	golang.org/x/net v0.4.0 // indirect
-	golang.org/x/sys v0.3.0 // indirect
-	golang.org/x/text v0.5.0 // indirect
+	golang.org/x/net v0.8.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
```

Comment:
Why is this not 0.6.0?

Reply:
x/net needed to be updated to >= 0.7.0 for a security issue.
go mod tidy update x/sys to 0.6.0 once x/net was updated.